### DISCUSSION TITLE
how to deploy html-css-js project at github?

### DISCUSSION THREAD

**Discussion by noahstoner** [community/community#134061] [GitHub Education] [answered]
### Hi everyone,

Hi, everyone
If you are new to github, you don't know about deploying project.
There are many ways to deply.
For example, vercel and netlify.
We can also deploy project at github.
Let's discuss about it.

**Comment by elsycharles** [comment]
Me personally I recommand Vercel for deployement 'cause Vercel doesn't have limit time not like another deployement.
you can deploy backend and frontend in it, I don't think we can deploy database but I'm finding a database deployement without limit if someone know about it

**Reply by noahstoner** [reply]
Thank you.
But do you know how to deploy using github.com?

**Reply by elsycharles** [reply]
If you wanna use gihub you can use gihub page [https://pages.github.com/](https://pages.github.com/) and it's free

**Comment by chivalrousdev** [comment, marked as answer]
Hi
Nice to meet you.
You can check the way at https://livecode247.com/deploying-a-static-html-and-css-site-using-github-pages
After creating new repository, in setting page of repository there is page option.
In there, you can find any option to deploy your project.
Wish you success

**Reply by noahstoner** [reply]
Thank you @chivalrousdev 
Do you have ever created with this way?

**Comment by noahstoner** [comment]
Thank you for your attention.
